code/game/objects/items/science_sampler.dm

```diff
@@ -0,0 +1,334 @@
+/*
+	Sampling
+*/
+/obj/item/sampler
+	name = "science sampler"
+	desc = "A multi-purpose sampling device designed by Zeng-Hu Pharmaceuticals for gathering samples during their research expeditions. It has attachments allowing for sampling of biological tissue, surface soil and water sources."
```

Comment:
`desc` is a bit long, could be split up into `desc` and `desc_extended`
but definitely needs `desc_info` explaining how to use it
see `/obj/structure/survey_probe` for an example